accelerator/common_core.c

```diff
@@ -402,6 +400,12 @@ int ta_find_transactions_obj_by_tag(const iota_client_service_t* const service,
   // get transaction obj
   for (hash_trits = hash243_queue_peek(hash_res->hashes); hash_trits != NULL;
        hash_trits = hash243_queue_peek(hash_res->hashes)) {
+    ta_get_transaction_object_res_t* obj_res =
+        ta_get_transaction_object_res_new();
+    if (obj_res == NULL) {
+      ret = -1;
```

Comment:
By the way, can we always use mnemonic return code instead of ordinary numbers?

Reply:
We will do this in #72 for sure.